forch/authenticator.py

```diff
@@ -54,8 +56,16 @@ def process_auth_result(self):
             auth_example = dict_proto(auth_obj, AuthResult)
             sys.stdout.write(str(proto_dict(auth_example)) + '\n')
 
+    def do_mab_request(self):
+        Socket = collections.namedtuple('Socket', 'listen_ip, listen_port, server_ip, server_port')
+        socket_info = Socket('0.0.0.0', 0, '172.17.0.5', 1812)
```

Comment:
as discussed, make command line arguments.  Seems like it should be the server_ip, the 'SECRET' mac address, and nonce.